<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenTUNG Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="installButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>87</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>87</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch OpenTUNG</string>
    </property>
   </widget>
   <widget class="QListWidget" name="worldList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenTUNG_Settings">
   <property name="text">
    <string>OpenTUNG Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
